options/locale/locale_en-US.ini

```
@@ -620,6 +620,17 @@ gpg_token_code = echo "%s" | gpg -a --default-key %s --detach-sig
 gpg_token_signature = Armored GPG signature
 key_signature_gpg_placeholder = Begins with '-----BEGIN PGP SIGNATURE-----'
 verify_gpg_key_success = The GPG key '%s' has been verified.
+ssh_key_verified=Verified Key
+ssh_key_verified_long=Key has been verified with a token and can be used to verify commits matching any activated email addresses for this user.
+ssh_key_verify=Verify
+ssh_invalid_token_signature = The provided SSH key, signature and token do not match or token is out-of-date.
```

```suggestion
ssh_invalid_token_signature = The provided SSH key, signature or token do not match or token is out-of-date.
```
Seems like it should be `or`, correct me if I'm wrong.